src/edit_session.rs

```diff
@@ -494,6 +494,22 @@ impl EditSession {
         }
     }
 
+    pub(crate) fn reverse_contours(&mut self) {
+        let mut path_ixs = BTreeSet::new();
+        for entity in &*self.selection {
+            if let Some(path_ix) = self.paths.iter().position(|p| p == entity) {
```

Comment:
this will reverse every path that has at least one selected point, is that okay?

Reply:
It matches the Glyphs behavior, and seems reasonable to me.